package.json

```diff
@@ -42,6 +43,7 @@
     "lodash": "^3.10.1",
     "lodash.debounce": "^3.1.1",
     "morgan": "^1.6.1",
+    "nodobjc": "TooTallNate/NodObjC#2.1.0",
```

Comment:
I didn't notice that there 2 different npm packages:
https://www.npmjs.com/package/NodObjC
https://www.npmjs.com/package/nodobjc